### DISCUSSION TITLE
How to add tag from the list in capture mode

### DISCUSSION THREAD

**Discussion by Zavrod** [chhoumann/quickadd#718] [Q&A]
Hello peeps,
noob here. As you'll later find out I'm not the sharpest tool in the drawer. 
I like the quickadd plugin a lot, but I'm loosing my mind cos I can't find an easy way how to capture task to my "Tasks" note which would add a tag to the same task from the list of all tags in Obsidian. 
I can manage to add the tag if I write it exactly right but I'm not exactly right all the time. (see part with the drawer)
Can someone help, please?
Ideal scenario would be:
1. hit the shortcut like it owes me money
2. prompt to add Task name
3. enter
4. prompt to write the tag and while I'm writing the list of tags getting smaller and helps me to choose the right one
5. enter
6. new line in my tasks note formated like this: "- [ ] New task #existingtagfromvault "
7. my life is great forever
I will be forever grateful to anyone who can save me from this misery :)
Cheers
Jan

**Comment by chhoumann** [comment]
Hi Jan - coming back to this very late, sorry. The good news: your ideal scenario works out of the box, no scripting needed.

1. Create a **Capture** choice targeting your Tasks note.
2. Enable **Capture format** and set the format to:

```
- [ ] {{VALUE:Task name}} {{VALUE:Tag}}
```

3. Assign your hotkey to the choice and hit it like it owes you money.

Each named `{{VALUE:...}}` opens its own prompt, so you get: task name prompt, Enter, tag prompt. In the tag prompt, type `#` - QuickAdd pops up a fuzzy suggester over every existing tag in your vault that narrows as you type. Pick one, Enter, and the full `#tag` lands in the line. That is your step 4 exactly, no drawer sharpness required.

If you want the tasks to land under a specific heading rather than at the top of the note, enable **Insert after** in the capture settings and point it at that heading.

The `#` trigger (plus `[[` for files) is described here: https://quickadd.obsidian.guide/docs/SuggesterSystem
